server/controllers/adminController.js

```diff
@@ -23,6 +23,20 @@ class AdminController {
             });
         }
     }
+
+    static getAllParties(req, res) {
+        try {
```

Comment:
Why are you making use of try-catch block here?

Reply:
Oh! It is not needed since I'm working with a dummy database